### ISSUE TITLE
[using pino-pretty] issue Cannot read property 'env' of undefined ==> const env = process["default"].env || {}

### ISSUE THREAD

**Issue by franckOL** [jorgebucaran/colorette#73]
This happens in file /colorette/index.cjs:6:32

**Comment by markrzen** [comment]
I am seeing this as well:

```sh
TypeError: Cannot read property 'env' of undefined
    at Object.<anonymous> (/app/node_modules/colorette/index.cjs:6:32)
     at Module._compile (internal/modules/cjs/loader.js:1072:14)
     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
     at Module.load (internal/modules/cjs/loader.js:937:32)
     at Function.Module._load (internal/modules/cjs/loader.js:778:12)
     at Module.require (internal/modules/cjs/loader.js:961:19)
     at require (internal/modules/cjs/helpers.js:92:18)
     at Object.<anonymous> (/app/node_modules/pino-pretty/index.js:3:30)
     at Module._compile (internal/modules/cjs/loader.js:1072:14)
     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
```

**Comment by markrzen** [comment]
@jorgebucaran You have an accidental breaking change in 2.0.6 with the rollup command changes.

**Comment by jorgebucaran** [comment]
Whoops, should be good now. Available in `2.0.7`.
